### PR TITLE
fix validation in lessons controller

### LINKED ISSUE #317: Fix validation on create and update lessons
Mentor or secrety can not create lessons  in future .

### PR DESCRIPTION
**Pull request by zelya2021**

### CODE REVIEW

File: CharlieBackend.Api.UnitTes/LessonServiceTests.cs [hunk 1]
```diff
@@ -141,8 +140,8 @@ public async Task CreateLessonAsync()
             StudentGroup studentGroup = new StudentGroup() { Id = 3, StudentsOfStudentGroups = studentOfStudentGroup };
             StudentGroup studentGroupWrong = new StudentGroup() { Id = 100 };
 
-            DateTime lessonDate = DateTime.Parse("2021-11-18T15:00:00.384Z");
-            DateTime lessonDateWrong = DateTime.Parse("2020-11-18T15:00:00.384Z");
+            DateTime lessonDate = DateTime.Parse("2020-11-18T15:00:00.384Z");
+            DateTime lessonDateWrong = DateTime.Parse("2021-11-18T15:00:00.384Z");
```
Comment (gingerbread-monster) [inline CharlieBackend.Api.UnitTes/LessonServiceTests.cs]: If I understand it correctly, for create/update lesson, `LessonDate > DateTime.Now` is considered invalid, right?
Then, judging by your changes to this line, this test will fail after 18th of December 2021

File: CharlieBackend.AdminPanel/appsettings.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   },
 
   "Cookies": {
-    "SecureKey": null
+    "SecureKey": "MyVerySecretKey"
```
Comment (gingerbread-monster) [inline CharlieBackend.AdminPanel/appsettings.json]: What is the purpose of this change and how is it related to this issue?

File: CharlieBackend.Api/appsettings.json [hunk 3]
```diff
@@ -7,11 +7,11 @@
     }
   },
   "AllowedHosts": "*",
-    "ConnectionStrings": {
-      "DefaultConnection": null,
-      "RabbitMQ": null,
-      "AzureBlobsAccessKey": null
-    },
+  "ConnectionStrings": {
+    "DefaultConnection":null,
```
Comment (gingerbread-monster) [inline CharlieBackend.Api/appsettings.json]:
```suggestion
    "DefaultConnection": null,
```
If you're removing redundant spaces then keep this one

**Comment by gingerbread-monster** [comment]
@QuestroL 
https://github.com/ita-social-projects/WhatBackend/issues/317#issue-788900638

Why a lesson with a `LessonDate` that is > `DateTime.Now` should be considered invalid? Isn't this how a mentor schedules a lesson for next day/week/whatever ?

**Comment by gingerbread-monster** [comment]
> _No description provided._

Please, always provide description of your changes.

**Review by QuestroL — APPROVED** [review]

**Review by Hlib-mats — APPROVED** [review]

**Review by gingerbread-monster — APPROVED** [review]

**Review by CptHouston — APPROVED** [review]

**Review by filinskyi — APPROVED** [review]

**Review by andy-dendy-repo — APPROVED** [review]

**Review by TetianaDomanska-dev — APPROVED** [review]
